lib/octopus_serverspec_extensions/type/octopus_deploy_environment.rb

```diff
@@ -48,10 +49,11 @@ def get_environment_via_api(serverUrl, apiKey, environment_name)
       environment = body['Items'].first unless body.nil?
     rescue => e
       puts "Unable to connect to #{url}: #{e}"
+      raise
```

Comment:
might be worth combining these 2 lines

Reply:
Actually - definitely combine them, as the exception thrown by `Net::HTTP.get_response` is pretty horrid and not very friendly

Reply:
done